src/components/dashboardV2/MembersList.vue

```diff
@@ -71,6 +71,7 @@ export default {
   },
   computed: {
     orderedMembers: function () {
+      console.log("Kasper");
```

Comment:
typo, sed s/Kasper/DAMNYOUKASPARNET :stuck_out_tongue_closed_eyes:

Reply:
Also, it should be: `console.error("DAMNYOUKASPARNET");`.

Perhaps add a `console.log("Kasper needs a beer");`